examples/sumo/grid.py

```diff
@@ -192,9 +194,9 @@ def grid_example(render=None, use_inflows=False):
         "maxDur": "6",
         "state": "ryryryryryry"
     }]
+    # ### TODO (kj) error here: need check for if the tl_logic your'e adding doesn't exist in the scenario
     tl_logic.add("center0", phases=phases, programID=1)
-    tl_logic.add("center1", phases=phases, programID=1)
-    tl_logic.add("center2", phases=phases, programID=1, tls_type="actuated")
+    # tl_logic.add("center2", phases=phases, programID=1, tls_type="actuated")
```

Comment:
Just add a check if there's more than 1 node for the center1, and more than 2 nodes for the center2

Reply:
Right now, the code flow replaces the old node with the new node, so if you tl_logic.add the same node multiple times, only the last will register. Is that okay?